foghorn-edge/files/logrotate_daemon.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
```

Comment:
License and Copyright information is missing.
open-ness/edgeapps repo hosts the open source edge applications. The license must be open source License.